<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirectWidget</class>
 <widget class="QWidget" name="DirectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>75</width>
    <height>66</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="Point_comboBox"/>
   </item>
   <item>
    <widget class="QPushButton" name="UPLINK_pushButton">
     <property name="text">
      <string>UPLINK</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="CANCEL_pushButton">
     <property name="text">
      <string>CANCEL</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
